Example/package.json

```diff
@@ -3,24 +3,33 @@
   "version": "1.0.0",
   "private": true,
   "scripts": {
-    "android": "react-native run-android",
-    "ios": "react-native run-ios",
+    "deviceList": "xcrun simctl list devices",
+    "setup": "npm i && npx jetify && cd ios && pod install && cd ..",
+    "android": "npx react-native run-android",
+    "ios": "npx react-native run-ios",
+    "build:ios": "react-native bundle --entry-file='index.js' --bundle-output='./ios/main.jsbundle' --dev=false --platform='ios'",
     "android:build": "cd ./android && ./gradlew assembleRelease && open './app/build/outputs/apk/'",
     "start": "node node_modules/react-native/local-cli/cli.js start",
     "test": "jest"
   },
   "dependencies": {
-    "react": "16.3.1",
-    "react-native": "0.55.4",
-    "react-native-animatable": "^1.3.0",
-    "react-native-indicators": "^0.13.0",
-    "react-native-spinner-button": "^1.0.1"
+    "react": "16.13.1",
+    "react-native": "0.63.4",
+    "react-native-animatable": "^1.3.3",
+    "react-native-indicators": "^0.17.0",
+    "react-native-spinner-button": "file:../react-native-spinner-button-1.0.4.tgz"
```

Comment:
This is a problem. It must not point to any zip. Point to the original github URL for the time being, or point to local directory. And once all changes are done and new version in published on npm, update this with a version number on npm

Reply:
Done